src/ontology/mondo-ingest.Makefile

```diff
@@ -553,12 +553,11 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
+# General
+update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
 
 EXTERNAL_CONTENT_DIR=external
 
-$(TMPDIR)/nord.tsv:
-	wget "https://rdbdev.wpengine.com/wp-content/uploads/mondo-export/rare_export.tsv" -O $@
-
 $(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
```

Comment:
## Error: `$(EXTERNAL_CONTENT_DIR)/mondo_genes.robot.owl`
- [ ] @matentzn @twhetzel Could use any advice here if you have any

Now that this is part of the normal `external/` pipeline, this goal gets run. I assume we may need it as well.

However, I'm getting a `MANCHESTER PARSE ERROR`:
```
MANCHESTER PARSE ERROR the expression ('has material basis in germline mutation in' some https://identifiers.org/hgnc/7763) at row 3, column 2 in table "external/mondo_genes.robot.tsv" cannot be parsed: encountered 'has material basis in germline mutation in', but expected one of:
	Class name
	Object property name
	Data property name
	inverse
	not
	( ... )
	{ ... }
```

<details><summary>Full log</summary>
<p>

```
sh run.sh make external-content-omim
mkdir -p external
wget "https://github.com/monarch-initiative/omim/releases/latest/download/mondo_genes.robot.tsv" -O external/mondo_genes.robot.tsv
--2024-06-06 22:50:34--  https://github.com/monarch-initiative/omim/releases/latest/download/mondo_genes.robot.tsv
Resolving github.com (github.com)... 140.82.112.4
Connecting to github.com (github.com)|140.82.112.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://github.com/monarch-initiative/omim/releases/download/2024-06-06/mondo_genes.robot.tsv [following]
--2024-06-06 22:50:35--  https://github.com/monarch-initiative/omim/releases/download/2024-06-06/mondo_genes.robot.tsv
Reusing existing connection to github.com:443.
HTTP request sent, awaiting response... 302 Found
Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/356336124/523db670-0628-41f1-972d-170112c4beb5?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20240606%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240606T225035Z&X-Amz-Expires=300&X-Amz-Signature=7dae072d881578a9a5f305abb07fd2b06a6852bab5f34015bfa85ba0899f886c&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=356336124&response-content-disposition=attachment%3B%20filename%3Dmondo_genes.robot.tsv&response-content-type=application%2Foctet-stream [following]
--2024-06-06 22:50:35--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/356336124/523db670-0628-41f1-972d-170112c4beb5?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20240606%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240606T225035Z&X-Amz-Expires=300&X-Amz-Signature=7dae072d881578a9a5f305abb07fd2b06a6852bab5f34015bfa85ba0899f886c&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=356336124&response-content-disposition=attachment%3B%20filename%3Dmondo_genes.robot.tsv&response-content-type=application%2Foctet-stream
Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.108.133, 185.199.110.133, 185.199.111.133, ...
Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 452703 (442K) [application/octet-stream]
Saving to: ‘external/mondo_genes.robot.tsv’

external/mondo_gene 100%[=================>] 442.09K  --.-KB/s    in 0.1s

2024-06-06 22:50:35 (4.05 MB/s) - ‘external/mondo_genes.robot.tsv’ saved [452703/452703]

robot --catalog catalog-v001.xml template \
	--template external/mondo_genes.robot.tsv \
	--prefix "orcid: https://orcid.org/" \
 	annotate \
			--ontology-iri http://purl.obolibrary.org/obo/mondo/external/nord.robot.owl \
			--version-iri http://purl.obolibrary.org/obo/mondo/external/2024-06-06/nord.robot.owl \
			-o external/mondo_genes.robot.owl
MANCHESTER PARSE ERROR the expression ('has material basis in germline mutation in' some https://identifiers.org/hgnc/7763) at row 3, column 2 in table "external/mondo_genes.robot.tsv" cannot be parsed: encountered 'has material basis in germline mutation in', but expected one of:
	Class name
	Object property name
	Data property name
	inverse
	not
	( ... )
	{ ... }
For details see: http://robot.obolibrary.org/template#manchester-parse-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make: *** [mondo-ingest.Makefile:562: external/mondo_genes.robot.owl] Error 1
Command exited with non-zero status 2
### DEBUG STATS ###
Elapsed time: 0:01.19
Peak memory: 174804 kb
```

</p>
</details> 

I've seen this error before, and it is addressed in the [ROBOT docs](https://robot.obolibrary.org/template). I tried a few things that didn't work, and I won't be able to look at it further until next week.

What I tried so far:
1.

Reply:
[James Overton helped me](https://obo-communitygroup.slack.com/archives/C01BTR3E8CU/p1718235991406229) solve this.

I tried a bunch of things first, but ultimately what I need to do was to add this line to `$(EXTERNAL_CONTENT_DIR)/%.robot.owl`:
`--input $(IMPORTDIR)/ro_import.owl \`